airbyte-server/src/main/java/io/airbyte/server/ServerApp.java

```diff
@@ -160,9 +119,9 @@ public void configure() {
             // https://stackoverflow.com/questions/35669774/jersey-custom-exception-mapper-for-invalid-json-string
             .register(JacksonJaxbJsonProvider.class);
 
-    // add filters
-    requestFilters.forEach(rc::register);
-    responseFilters.forEach(rc::register);
+    // inject custom server functionality
+    customComponentClasses.forEach(rc::register);
```

Comment:
what is the difference between `componentClass` and `component`? confused what to pass into what if we are starting up custom resources.

Reply:
See above. It depends on if it's a shared class or spun up for requests.